.evergreen/run-tox.sh

```diff
@@ -39,7 +39,7 @@ if [ "${INSTALL_TOX}" = "true" ]; then
         source motorenv/Scripts/activate
     fi
     set -o xtrace
-    pip install tox
+    pip install tox>=3.18
```

Comment:
Doesn't this just give us the latest version? Whats the reason you pinned this?

Reply:
Because if an earlier version of tox is already installed ` tox>=3.18` will upgrade it but `tox` will not. allowlist_externals was added in tox 3.18